### PR TITLE
Add note functionality

### PR DESCRIPTION
**Pull request by SanketDhabarde**
### Changes made
- Add the note
- pin the note
- displayed on the home page pinned/unpinned

### Preview link
[Preview](https://deploy-preview-8--always-notes.netlify.app/)

### CODE REVIEW

File: src/components/CreateNote/CreateNote.css [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+.note-input {
+  background: transparent;
+  color: var(--primary-text);
+  border: none;
+  width: 100%;
+  border-radius: 5px;
+  outline: none;
+}
+
+textarea {
```
Comment (anupkgurung) [inline src/components/CreateNote/CreateNote.css]: do not use  style on element.

File: src/components/CreateNote/CreateNote.jsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+import React, { useReducer } from "react";
+import { v4 as uuid } from "uuid";
+import { useNotes } from "../../context/note-context";
```
Comment (anupkgurung) [inline src/components/CreateNote/CreateNote.jsx:3]: try to use barrel export

File: src/components/Nav/Nav.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
+.nav {
+  display: flex;
+  font-size: 1.2rem;
+  width: 100%;
+  height: 6vh;
+  border-top-right-radius: 20px;
```
Comment (anupkgurung) [inline src/components/Nav/Nav.css]: use rem instead of px

File: src/components/Note/Note.css [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+.note-icon {
+  width: 1rem;
+  height: 1rem;
+  padding: 1rem;
+}
+
+.note-icon:hover {
+  background-color: rgba(0, 0, 0, 0.6);
```
Comment (anupkgurung) [inline src/components/Note/Note.css:29]: try to use hex code instead, also put colors in variables.

**Review by anupkgurung — APPROVED** [review]
left few suggestion

**Review by jeetbhalani15 — APPROVED** [review]
- Few suggestions as @anupkgurung said.
- Pin notes & its working nicely done.
- All Notes changes are working fine.

File: src/components/CreateNote/CreateNote.jsx (hunk 2 above)
Comment (SanketDhabarde) [reply src/components/CreateNote/CreateNote.jsx:3]: I will do this when I have more context. thanks for suggestion

**Pull request merged**